plugins/woocommerce/src/Blocks/BlockPatterns.php

```diff
@@ -119,6 +120,7 @@ public function register_block_patterns() {
 			'keywords'      => 'Keywords',
 			'blockTypes'    => 'Block Types',
 			'inserter'      => 'Inserter',
+			'featureFlag'   => 'Feature Flag',
```

Comment:
Hey, @albarin! 👋  

Pinging you since you've wrote the PR that added headers https://github.com/woocommerce/woocommerce-blocks/pull/6861. I've added another header to add/remove block pattern based on feature flag. I've tested this quite extensibly, but please let me know if you have any concerns in my approach, thanks!